### DISCUSSION TITLE
Routing issue when published to IIS but runs fine under kestrel

### DISCUSSION THREAD

**Discussion by erehtolleh** [serenity-is/Serenity#5542] [Help] [answered]
I am using .Net core 5  of Serenity and when I test the project, all the navigation links works.  When I publish the project to IIS to a different server, I get a 404 error when trying to navigate to any of the pages.  It can't seem to find the correct route of the page.  If I run on the same published files using Kestrel, everything runs fine. 

Does anyone know what settings under IIS or configurations in order to fix this issue?

**Comment by erehtolleh** [comment, marked as answer]
My navigationmodel.cs file was off from the sample project.

**Comment by hafizTahir** [comment]
@erehtolleh 

same issue did you find any solution ?
